app/lib/efile/nj/nj_retirement_income_helper.rb

```diff
@@ -60,11 +92,18 @@ def spouse_62_and_older?
         @intake.calculate_age(@intake.spouse_birth_date, inclusive_of_jan_1: false) >= 62 
       end
 
-      def line_28b_eligible?(line_15, line_27, line_28a)
+      def all_filers_under_62?
         if @intake.spouse_birth_date.present?
           spouse_or_primary_is_age_eligible = spouse_62_and_older? || primary_62_and_older?
-          return false unless spouse_or_primary_is_age_eligible
+          return true unless spouse_or_primary_is_age_eligible
         elsif !primary_62_and_older?
+          return true
+        end
+        false
+      end
+
+      def line_28b_eligible?(line_15, line_27, line_28a)
+        if all_filers_under_62?
           return false
         end
```

Comment:
[dust] for cleanliness I like the pattern on the line below `return false if ...` instead of the three-line if statement here